src/subscription_manager/injection.py

```diff
@@ -81,7 +81,7 @@ def require(self, feature, *args, **kwargs):
 
         if isinstance(provider, (type, six.class_types)):
             self.providers[feature] = provider(*args, **kwargs)
-        elif six.callable(provider):
+        elif callable(provider):
```

Comment:
It looks like that it was last usage of `six` in this file. Can you please remove importing of this module<